README.md

```diff
@@ -24,10 +24,11 @@ Each example is built from 3 files:
 
 - `<example-id>.edifact.dat` - message provided by laboratory
 - `<example-id>.fhir.json` - message translated to FHIR format
-- `<example-id>.nhsack.dat` - confirmation messasge sent back to the laboratory
 
-Pathology examples can be found here //TODO
-Screening examples can be found here //TODO
+Example input EDIFACT files can be found [here](https://github.com/nhsconnect/integration-adaptor-lab-results/tree/main/src/intTest/resources/edifact) with the corresponding FHIR translations found [here](https://github.com/nhsconnect/integration-adaptor-lab-results/tree/main/src/intTest/resources/fhir)
+
+Pathology examples can be found [here](https://github.com/nhsconnect/integration-adaptor-lab-results/tree/main/src/intTest/resources/success_uat_data/NHS003)
+Screening examples can be found //TODO
```

Comment:
Are we leaving this TODO for the future, or can it be addressed now?

Reply:
We don't have test data for Screening prepared yet, so this will have to be left until later.